bioblend/galaxy/objects/client.py

```diff
@@ -16,7 +16,6 @@
 
 @six.add_metaclass(abc.ABCMeta)
 class ObjClient(object):
-
```

Comment:
Restore blank line here and at line 83.

Reply:
Fixed by https://github.com/galaxyproject/bioblend/pull/247/commits/fc4451f610bc695cb0c4d11f6b97f115baa31bd9